docs/release-notes/release-notes-1.6.0-rc1.1.md

```diff
@@ -0,0 +1,925 @@
+# dcrd v1.6.0-rc1.1
+
+This release of dcrd introduces a large number of updates.  Some of the key
+highlights are:
+
+* A new consensus vote agenda which allows the stakeholders to decide whether or
+  not to activate support for a decentralized treasury
+* Aggregate fee transaction selection in block templates (CPFP)
+* Improved peer discovery via HTTPS seeding with filtering capabilities
+* Major performance enhancements for signature validation and other
+  cryptographic operations
+* Approximately 15% less overall resident memory usage
+* Proactive signature cache eviction
+* Improved support for single-party Schnorr signatures
+* Ticket exhaustion prevention
+* Various updates to the RPC server such as:
+  * A new method to retrieve the current treasury balance
+  * A new method to query treasury spend transaction vote details
+* Infrastructure improvements
+* Quality assurance changes
+
+For those unfamiliar with the voting process in Decred, all code needed in order
+to support a decentralized treasury is already included in this release, however
+it will remain dormant until the stakeholders vote to activate it.
```

Comment:
If we add a context para for newcomers you might also want to add

> The voting process is described in detail [here](https://docs.decred.org/governance/consensus-rule-voting/overview/).

or simply put that link on the `voting process`

Reply:
Added link to voting process.